magenta-lib/src/main/scala/magenta/input/models.scala

```diff
@@ -0,0 +1,55 @@
+package magenta.input
+
+import play.api.libs.json.{JsValue, Json, Reads}
+
+
+case class RiffRaffDeployConfig(
```

Comment:
There's nothing yaml specific about this class, so I've proposed changing its name. Not particularly wedded to this one, suggestions welcome.